ibm/structures.go

```
@@ -137,3 +139,64 @@ func flattenServiceInstanceCredentials(keys []mccpv2.ServiceKeyFields) []interfa
 	}
 	return out
 }
+
+func flattenIAMPolicyResource(list []iampapv1.Resources, iamClient iampapv1.IAMPAPAPI) []map[string]interface{} {
+	result := make([]map[string]interface{}, 0, len(list))
+	for _, i := range list {
+		name := i.ServiceName
+		var serviceName string
+		if name != "" {
+			serviceName, _ = iamClient.IAMService().GetServiceDispalyName(name)
```

Are we ignoring the error here?